makefile

```diff
@@ -31,7 +30,8 @@ include make/libraries
 # Set default compiler options.
 ##
 CFLAGS = -I . -isystem $(EIGEN) -isystem $(BOOST) -isystem$(CVODES)/include -Wall -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DNO_FPRINTF_OUTPUT -pipe
-CFLAGS_GTEST = -DGTEST_USE_OWN_TR1_TUPLE
+CXXFLAGS = -std=c++14
```

Comment:
We're actually using `CFLAGS` and `CC` to build C++ code (as I mentioned before, it's a misuse of make, but we've been doing it since the start and there wasn't a great time to shift it to the proper use of make).

It looks like `CXXFLAGS` is only used for testing. (line 34 of makefile) Any reason not to just add `-std=c++14` directly to `CFLAGS_GTEST`? Just wanting to figure out the purpose of the indirection.